demos/can-component/accordion.html

```diff
@@ -75,19 +75,21 @@
 
 Component.extend({
 	tag: "my-accordion",
-	ViewModel: AccordionViewModel,
-	leakScope: true
```

Comment:
I’m not sure how `leakScope` is supposed to work in 4.0, but it was really buggy in this demo.